assets/js/googlesitekit/datastore/user/permissions.test.js

```diff
@@ -481,6 +565,50 @@ describe( 'core/user authentication', () => {
 
 				expect( canViewSharedModule ).toBe( true );
 			} );
+
+			it( 'should change module slug to "analytics-4" when the `module.slug` is "analytics" and the dashboard view is GA4', () => {
```

Comment:
This test is good, but I'd suggest rewording the description a bit to describe it more in terms of the outcome as opposed to the underlying implementation:

```suggestion
			it( 'should treat `analytics` as `analytics-4` when the dashboard view is GA4', () => {
```

Reply:
Updated ✅